static/styles/portico/portico.css

```diff
@@ -1717,19 +1717,19 @@ label.label-title {
         font-size: 12px;
         color: hsl(0, 50%, 60%);
     }
-
-    .api-argument-datatype {
-        text-transform: lowercase;
-        font-weight: 600;
-        font-size: 14px;
-        color: hsl(176, 46.4%, 41%);
-    }
 }
 
-.api-response-datatype {
+.api-request-parameter-datatype,
+.api-response-parameter-datatype {
     color: hsl(176, 46.4%, 41%);
 }
 
+.api-request-parameter-datatype {
+    text-transform: lowercase;
+    font-weight: 600;
+    font-size: 14px;
+}
+
```

Comment:
Why not just have a single `.api-field-type` common class name used for rendering the data types in both contexts, without reference to request to response (since I imagine that detail doesn't affect the formatting)?  

Or is there a font size difference here you're trying to emphasize?  If so, I still think that'd be better as extra sizing CSS for the `.api-argument .api-field-type` selector.

Reply:
(What I'm trying to get away from is parallel CSS with semi-duplicate names for a single concept).

Reply:
@timabbott Made the required changes